content/monitors/notifications.md

```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
+{{< img src="monitors/notifications/templateconditionaltrigger.png" alt="template conditional trigger" responsive="true" popup="true">}}
 
-### Include triggering tags in notification title
+and the recovery notification:
 
-In order to save some space, you can Include triggering tags in notification title. This makes your notification title look like this:
+{{< img src="monitors/notifications/templateconditionalrecover.png" alt="template conditional recover" responsive="true" popup="true" >}}
 
-{{< img src="monitors/notifications/templatevar_short.png" alt="template var short" responsive="true" popup="true" >}}
+#### `{{is_recovery}}` or `{{is_alert_recovery}}` 
 
-Note that template variable content is escaped by default. If your variable
-contains JSON or code that you would NOT like to be escaped, then use triple braces instead of double braces (e.g. `{{{event.text}}}`).
+* `{{is_recovery}}` triggers and a monitor recovers indifferently either from a **WARNING** state or an **ALERT** state. 
+* `{{is_alert_recovery}}` triggers when a monitor recovers directly from an **ALERT** state to **OK** state. 
+* `{{is_warning_recovery}}` triggers when a monitor recovers from a **WARNING** state to **OK** state
+
+This means that if the monitor switches from ALERT to WARNING then OK state:
+
+* the `{{is_recovery}}` would trigger
+* the `{{is_alert_recovery}}` wouldn't trigger 
+* the `{{is_warning_recovery}}` would trigger. 
+
+The @ notification inside the template variables follows the same rules. 
 
-### Conditional variables for different triggering scopes
-You can have a monitor event display a different message depending on the group that's causing a notification.
+#### `{{is_match}}`
 
-The `{{is_match}}` conditional lets you match the triggering context to some
-string. For example, you might want to notify your db team if a triggering
-host has `role:db` but notify your app team if the host has `role:app`.
+The `{{is_match}}` conditional allows you to match the triggering context to some string to display a different message.  
+For example, you might want to notify your db team if a triggering host has `role:db` but notify your app team if the host has `role:app`.
 
 You can use any of the available tag variables in your condition. A match
 is made if the comparison string is anywhere in the resolved variable.
 
 The variable uses the following format:
 ```
-  {{#is_match "tag_variable" "comparison_string"}}
+{{#is_match "tag_variable" "comparison_string"}}
   This shows if comparison_string is in tag_variable.
-  {{/is_match}}
+{{/is_match}}
 ```
 
 Here is an example of how you can give a different message depending on the triggering context:
 
 {{< img src="monitors/notifications/scope_match_editor.png" alt="scope match editor" responsive="true" popup="true">}}
 
-## Variable availability
+**Note**: To use `{{is_match}}` conditional to check if a `tag_variable` is **NOT** empty, append `.name` after your tag name, for instance:  
```

Comment:
Could you please add the word "the" in front of `{{is_match}}` like:

```md
**Note**: To use the `{{is_match}}` conditional to 
```

Reply:
👍